lib/types/src/conflict.ts

```diff
@@ -1,4 +1,4 @@
-import type { Plugin } from '@dotcom-tool-kit/types'
+import type { Plugin } from './'
```

Comment:
**nitpick**: I hate this

```suggestion
import type { Plugin } from './index'
```

Reply:
Please could you elaborate @ivomurrell?

Reply:
I don't think it's particularly readable

Reply:
I thought it was self-evidently unreadable. If people disagree I'm fine to keep it as it's just a nitpick

Reply:
I do prefer to be verbose and prefer the change you have suggested. @apaleslimghost are you okay with committing the requested change?